api-report/merge-tree.api.md

```diff
@@ -187,6 +187,7 @@ export class Client {
     peekPendingSegmentGroups(count?: number): SegmentGroup | SegmentGroup[] | undefined;
     posFromRelativePos(relativePos: IRelativePosition): number;
     rebasePosition(pos: number, seqNumberFrom: number, localSeq: number): number;
```

Comment:
not pictured here so commenting on a random line, but you should update the comment in `handleToPosition` which talks about the need to walk all segments on reconnect (since now it's only used for undo-redo).